common/lib/client/realtimepresence.js

```diff
@@ -259,6 +258,10 @@ var RealtimePresence = (function() {
 	};
 
 	PresenceMap.prototype.put = function(item) {
+		if(Utils.arrIn([presenceAction.ENTER, presenceAction.UPDATE], item.action)) {
```

Comment:
This might look elegant but it's hopelessly inefficient compared to just doing (x==a || x==b).